script/update

```diff
@@ -237,7 +237,7 @@ def gn_args(target_arch, component, gyp_defines):
   if component == 'shared_library':
     args.append('is_component_build=true')
   else:
-    args.append('is_component_build=false is_official_build=true')
+    args.append('is_component_build=false')
```

Comment:
This seems like it could potentially degrade performance. Out of curiosity, how was the libv8_base library corrupted?